ups/ap_verify.cfg

```diff
@@ -1,12 +1,12 @@
-# -*- python -*-
-
-import lsst.sconsUtils
-
-dependencies = {
-}
-
-config = lsst.sconsUtils.Configuration(
-    __file__,
-    hasDoxygenInclude=False,
-    hasSwigFiles=False,
+# -*- python -*-
+
+import lsst.sconsUtils
+
+dependencies = {
+}
+
+config = lsst.sconsUtils.Configuration(
+    __file__,
+    hasDoxygenInclude=False,
+    hasSwigFiles=False,
```

Comment:
I don't see anything that changed here. Is this just something weird the commit history?

Reply:
That's the "Fix Windows line endings" commit.